### ISSUE TITLE
[VoiceOver] Post Settings concerns

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12958]
<img src="https://user-images.githubusercontent.com/198826/68971251-588fb500-07a6-11ea-883c-5cfa9943f55c.png" width="320">

## Main

- [ ] _Set Featured Image_ is not a button
- [ ] _Set Location_ is not a button
- [ ] If there is a location and a map is shown, the map pin and other elements are accessible. They don't need to be and it's probably not that useful. 

## Post Categories

- [ ] The _Add_ button's label is confusing: “icon post add”
- [ ] Cells do not have the `.button` trait

## Add A Category

- [ ] Parent Category selection -> Cells do not have the `.button` trait

## Tags

- [ ] The tag cells do not have `.button` trait
- [ ] When selecting a suggested tag, the focus goes to the text editor but this is not announced

## Published On

- [ ] The “Date and Time” cell can be a `.button`

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
